### PR TITLE
Adding in Pull request before Halloween!!!!!

### PR DESCRIPTION
**Pull request by FusionMusic**
Practicing adding in a pull request the week of Halloween!!

### CODE REVIEW

**Comment by pjc667** [comment]
Just in time then

File: inflammation/models.py
Comment (pjc667) [inline inflammation/models.py:1]: Looks ok, but it's been a long time since A Level Maths

File: inflammation/models.py [hunk 1]
```diff
@@ -54,3 +54,13 @@ def daily_min(data):
...
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
```
Comment (pjc667) [inline inflammation/models.py:65]: Need to test for zero length data. Could get a divide by zero here
